rcl/test/rcl/test_subscription.cpp

```diff
@@ -1396,6 +1396,149 @@ TEST_F(TestSubscriptionFixtureInit, test_subscription_bad_take) {
   rcl_reset_error();
 }
 
+TEST_F(TestSubscriptionFixture, test_subscription_option_ignore_local_publications) {
+  // The current Implementations of DDS for ignoring local publications
+  // While creating a subscription, the ignore_local_publications option is set to true
+  //                                               Notification of message
+  //            Connected to local publications    from local publication      Take data
+  //            -------------------------------    -----------------------      ---------
+  // Fastdds               Yes                               Yes                   No
```

Comment:
How about using rmw name instead?

```suggestion
  // rmw_fastrtps               Yes                               Yes                   No
```

Reply:
Okay